pylint/config/config_initialization.py

```diff
@@ -55,23 +55,37 @@ def _config_initialization(
 
     # Load command line arguments
     try:
-        args_list = linter.load_command_line_configuration(args_list)
+        parsed_args_list = linter.load_command_line_configuration(args_list)
     except SystemExit as exc:
         if exc.code == 2:  # bad options
             exc.code = 32
         raise
 
     # args_list should now only be a list of files/directories to lint. All options have
     # been removed from the list
-    if not args_list:
+    if not parsed_args_list:
         print(linter.help())
         sys.exit(32)
 
     # We have loaded configuration from config file and command line. Now, we can
     # load plugin specific configuration.
     linter.load_plugin_configuration()
```

Comment:
Why is this still needed (given that it was moved below)?

Reply:
I think you're referring to `linter._parse_plugin_configuration()`?

That has not been implemented yet. The code below is just the order in which things should occur. This is "captured" by `# TODO: Start of the implemenation of argument parsing with argparse` and the `TODO` in the method definition.